src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs

```diff
@@ -142,7 +142,7 @@ private async Task<T> SearchPage<T>(
                 {
                     var types = string.Join("&",
                         filters.PackageTypes.Select(
-                            s => "packageTypeFilter=" + s));
+                            s => "packageType=" + s));
```

Comment:
The fact that no tests failed, shows a gap in testing. As per the PR checklist, please add tests.

There's already a file [`PackageSearchResourceV3Tests.cs`](https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.Tests/NuGet.Protocol.Tests/PackageSearchResourceV3Tests.cs) that shows a pattern how to test with mock http responses.